web-components/libs/map/src/lib/actions/map/map-with-actions.spec.ts

```diff
@@ -112,71 +104,13 @@ describe('map with actions', () => {
         expect(activateDefaultActionSpy).toHaveBeenCalled();
     });
 
-    it('there are 9 predefined interactions', () => {
```

Comment:
Test voor mij de behaviour van de OL MAP... https://openlayers.org/en/latest/apidoc/module-ol_interaction_defaults#:~:text=The%20default%20set%20of%20interactions,DragPan